ehsm_kms_service/ehsm_kms_params.js

```diff
@@ -0,0 +1,74 @@
+
+const ehsm_keyspec_t = {
+  EH_AES_GCM_128: 0,
+  EH_AES_GCM_256: 1,
+  EH_RSA_2048: 2,
+  EH_RSA_3072:3,
+  EH_EC_P256: 4,
+  EH_EC_P512: 5,
+  EH_EC_SM2: 6,
+  EH_SM4: 7,
+}
+const ehsm_keyorigin_t = {
+  EH_INTERNAL_KEY: 0,
+  EXTERNAL_KEY: 1,
+}
+const keyspec= {
+  type : 'const',
+  arr: Object.keys(ehsm_keyspec_t),
+  errortext: "The keyspec type is incorrect",
+};
+
```

Comment:
keep the define align with "origin"
const keyspec= {
  type : 'const',
  arr: Object.keys(ehsm_keyspec_t),
  errortext: "The keyspec type is incorrect",
};